packages/vscode-extension/src/builders/PlatformBuildCache.ts

```diff
@@ -112,16 +112,18 @@ export class PlatformBuildCache {
   }
 
   public async calculateFingerprint() {
+    Logger.info("Calculating fingerprint");
```

Comment:
Why is this log not debug level?

Reply:
we rarely use Logger.debugg as currently we redirect almost all logs to info anyway to make it easier for us to trace down reported issues

Reply:
That's is true that we redirect most logs to info but we do it on the logger level in `Logger.ts` file the code looks like this: 

```javascript
debug(message: string, ...args: any[]) {
    // in the initial phase od development, we want to surface debug messages such that
    // they can be seen by the extension user. This is useful for collecting feedback and
    // being able to better troubleshoot issues. In vscode, the default logging level is set
    // to INFO. We don't want to ask users to change their logging level as it impacts all
    // the logs vscode generates. Also, when changing log level, one need to reload vscode
    // window for the extension to pick up the change.
    outputChannel.info(message, ...args);
  },
```

Eventually those logs do not seem info level as they do not provide information to a user that is not familiar with our code base, so I would mark the as `debug`

Reply:
If I had this logs I would've fixed this problem long time ago as they'd result in an excessive number of entries getting printed out. I don't really care whether its debug or info since they ultimately result in the same behavior at the moment, so not going to insist to keep it one way of the other, but I don't agree they don't provide any information as at least they would've prevented the issue I'm fixing here